src/common/Configuration.h

```diff
@@ -37,6 +37,9 @@ namespace SDDM {
         enum NumState { NUM_NONE, NUM_SET_ON, NUM_SET_OFF };
 
         //  Name                   Type         Default value                                   Description
+        // TODO: In SDDM 2.0 we want x11-user to be the default
```

Comment:
We don't even have SDDM 1.0 yet, what milestone is this TODO aimed for?

Reply:
The plan was to merge this PR and #1379  and then release SDDM 1.0.
Do a few minor release and release SDDM 2.0.
For 2 reasons: it's time for SDDM 1.0 and I didn't feel like breaking the workflow now.

Reply:
I guess. I think you should switch it for SDDM 1.0, but otherwise it makes sense.